docs/_data/menu-sql.yaml

```diff
@@ -123,37 +123,27 @@
             - text: SELECT 
               url: sql-ref-syntax-qry-select.html 
               subitems:
-                - text: DISTINCT Clause
-                  url: sql-ref-syntax-qry-select-distinct.html
```

Comment:
I see, so these don't exist anymore?

Reply:
@srowen Yeah Sean. So when we setup this initially, i had created placeholder links for the topics i thought we should cover. As you pointed out, it's not a good idea to have these dead links. So i have removed them. I am writing "FROM clause" and "WINDOW clause" now. With that, i think we will have most of the clauses covered. If we think we need to take on some specific topic an explain in more detail then, we can add these along with the links. what do you think ?